sycl/include/sycl/ext/intel/esimd/memory.hpp

```diff
@@ -1353,9 +1353,32 @@ void simd_obj_impl<T, N, T1, SFINAE>::copy_to(
       if constexpr (RemN == 1) {
         Addr[NumChunks * ChunkSize] = Tmp[NumChunks * ChunkSize];
       } else if constexpr (RemN == 8 || RemN == 16) {
-        simd<uint32_t, RemN> Offsets(0u, sizeof(T));
-        scatter<UT, RemN>(Addr + (NumChunks * ChunkSize), Offsets,
-                          Tmp.template select<RemN, 1>(NumChunks * ChunkSize));
+        // TODO: Remove once fixed on GPU side
```

Comment:
Looks good/correct. Please only make this comment a bit more explanatory and specific what part of the code below and why should be removed. E.g.:
```suggestion
        // TODO: GPU runtime may handle scatter of 16 byte elements incorrectly.
        // The code below is a workaround which must be deleted once GPU runtime is fixed.
```